ulib/LowStar.Regional.fst

```diff
@@ -0,0 +1,79 @@
+module LowStar.Regional
+
+open FStar.All
+open FStar.Integers
+open FStar.Classical
+open LowStar.Modifies
+
+module HH = FStar.Monotonic.HyperHeap
+module HS = FStar.HyperStack
+module HST = FStar.HyperStack.ST
+module MHS = FStar.Monotonic.HyperStack
```

Comment:
Why distinguish `MHS` from `HS`? The latter includes the former

Reply:
Yes, no specific reason for this one; do you want me to fix it through all related files?

Reply:
Fixed in the below commit!